core/deployment/src/main/java/io/quarkus/deployment/util/JandexUtil.java

```diff
@@ -19,9 +20,14 @@
 import org.jboss.jandex.Type.Kind;
 import org.jboss.jandex.TypeVariable;
 
+import io.quarkus.builder.BuildException;
+
+/**
+ * A collection of Jandex utility methods.
+ */
 public final class JandexUtil {
 
-    private static final DotName OBJECT = DotName.createSimple(Object.class.getName());
+    public final static DotName DOTNAME_OBJECT = DotName.createSimple(Object.class.getName());
```

Comment:
I wonder why `org.jboss.jandex.DotName.OBJECT_NAME` is not public...

Reply:
Very good question. Perhaps @n1hility knows?